share/es.po

```diff
@@ -0,0 +1,285 @@
+"local puede causar mensajes de error confusos."
+
+#, perl-format
+msgid ""
+"Warning: setting locale category LC_MESSAGES to %s failed (is it installed "
+"on this system?).\n"
+"\n"
+msgstr ""
+"Advertencia: fallo en definir la categoría de localización LC_MESSAGES como "
+"%s (¿está instalada en este sistema?).\n"
+"\n"
+
+#, perl-format
+msgid ""
+"Warning: setting locale category LC_CTYPE to %s failed (is it installed on "
+"this system?).\n"
+"\n"
+msgstr ""
+"Advertencia: fallo en definir la categoría de localización LC_CTYPE como %s "
+"(¿está instalada en este sistema?).\n"
+"\n"
+
+#, perl-brace-format
+msgid "Loading profile from {path}."
+msgstr "Cargando perfil desde {path}."
+
+msgid "Failed to recognize stop level '"
+msgstr "Fallo en identificar el nivel de detención '"
+
+msgid ""
+"--level must be one of CRITICAL, ERROR, WARNING, NOTICE, INFO, DEBUG, DEBUG2 "
+"or DEBUG3.\n"
+msgstr ""
+"--level debe ser alguno entre CRITICAL, ERROR, WARNING, NOTICE, INFO, DEBUG, "
+"DEBUG2 o DEBUG3.\n"
+
+msgid "Must give the name of a domain to test.\n"
+msgstr "Debe indicar el nombre de un dominio a probar.\n"
+
+msgid "Seconds "
+msgstr "Segundos "
+
+msgid "Level     "
+msgstr "Nivel     "
+
+msgid "Module       "
+msgstr "Módulo       "
+
+msgid "Testcase       "
+msgstr "Caso de prueba       "
+
+msgid "Message"
+msgstr "Mensaje"
+
+msgid "======= "
+msgstr "======= "
+
+msgid "========= "
+msgstr "========= "
+
+msgid "============ "
+msgstr "============ "
+
+msgid "============== "
+msgstr "============== "
+
+msgid "======="
+msgstr "======="
+
+msgid "Looks OK."
+msgstr "Se ve bien."
+
+msgid ""
+"\n"
+"\n"
+"   Level\tNumber of log entries"
+msgstr ""
+"\n"
+"\n"
+"   Nivel\tNúmero de registros"
+
+#, perl-format
+msgid "%8s\t%5d entries.\n"
+msgstr "%8s\t%5d registros.\n"
+
+msgid ""
+"Warning: Zonemaster::LDNS not compiled with libidn, cannot handle non-ASCII "
+"names correctly."
+msgstr ""
+"Advertencia: Zonemaster::LDNS no fue compilado con libidn, no puede manejar "
+"correctamente nombres no-ASCII."
+
+msgid "DEBUG"
+msgstr "DEPURACIÓN"
+
+msgid "INFO"
+msgstr "INFO"
+
+msgid "NOTICE"
+msgstr "AVISO"
+
+msgid "WARNING"
+msgstr "ADVERTENCIA"
+
+msgid "ERROR"
+msgstr "ERROR"
+
+msgid "CRITICAL"
+msgstr "CRÍTICO"
+
+#~ msgid "The minimum severity level to display"
+#~ msgstr "El nivel mínimo de gravedad que despliega"
+
+#~ msgid "Name of configuration file to load."
+#~ msgstr "Nombre del archivo de configuración a cargar."
+
+#~ msgid "Loading configuration from {path}."
+#~ msgstr "Cargando configuración desde {path}."
+
+#~ msgid "Print the effective configuration used in JSON format, then exit."
+#~ msgstr ""
+#~ "Despliega la configuración definitiva usada en formato JSON, luego "
+#~ "termina."
+
+#~ msgid ""
+#~ "Local IP address that the test engine should try to send its requests "
+#~ "from."
+#~ msgstr ""
+#~ "Dirección IP local que debería usar el motor de pruebas como origen de "
+#~ "sus solicitudes."
```

Comment:
Would it be safe to remove these lines?

Reply:
Yes. It is safe both to keep and remove. If we keep them now I will check all PO files after the v2021.2.1 release and create a clean-up PR. Is that OK?

Reply:
It's fine for me to defer the removal (as long as we don't forger about it :) )

Reply:
I will not. @pnax, please approve.